browser/ui/views/frame/brave_browser_frame.cc

```diff
@@ -58,3 +62,16 @@ views::internal::RootView* BraveBrowserFrame::CreateRootView() {
   root_view_ = new BraveBrowserRootView(browser_view_, this);
   return root_view_;
 }
+
+void BraveBrowserFrame::SetTabDragKind(TabDragKind kind) {
+  const bool should_sync_shared_pinned_tab =
+      base::FeatureList::IsEnabled(tabs::features::kBraveSharedPinnedTabs) &&
+      tab_drag_kind_ == TabDragKind::kAllTabs && kind == TabDragKind::kNone;
+
+  BrowserFrame::SetTabDragKind(kind);
+
```

Comment:
nit:
```suggestion
  BrowserFrame::SetTabDragKind(kind);

  const bool should_sync_shared_pinned_tab =
      base::FeatureList::IsEnabled(tabs::features::kBraveSharedPinnedTabs) &&
      tab_drag_kind_ == TabDragKind::kAllTabs && kind == TabDragKind::kNone;

```

Reply:
Shouldn't we check the old value(tab_drag_kind_) before calling Browser::SetTabDragKind(), which sets the new value to the member variable?

Reply:
Ah, I see. it's set by `BrowserFrame::SetTabDragKind()`.
Just suggested to move vars closely where it's used. but it shouldn't in this case.